environment.yml

```
@@ -24,5 +24,6 @@ dependencies:
   - snpeff=4.3.1t
   - strelka=2.9.10
   - tiddit=2.7.1
+  - trim-galore=0.6.5
```

Just remembered that you need `pigz` in the environment too in order to use `TrimGalore` on multiple cores:
https://github.com/nf-core/atacseq/blob/3541076c2dc5a387c4b978e87c4ed8276a4cf5f8/environment.yml#L25

---

```suggestion
  - trim-galore=0.6.5
  - conda-forge::pigz=2.3.4 
```
I think this should do the trick